addons/ai/fnc_waypointGarrison.sqf

```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 /* ----------------------------------------------------------------------------
-Script: fnc_waypointGarrison
+Internal Script: CBA_fnc_waypointGarrison
```

Comment:
```suggestion
Internal Script: fnc_waypointGarrison
```
It's not compiled with prefix.

Reply:
Yeah, I just noticed it, fixing it rn along with some other minor errors.

Reply:
Bump ( ͡° ͜ʖ ͡°)